components/layout/index.js

```diff
@@ -1,7 +1,7 @@
 import Head from 'next/head'
-import Header from '../components/Header'
-import Footer from '../components/Footer'
-import { styled } from '../stitches.config.ts';
+import Header from './Header'
+import Footer from './Footer'
+import { styled } from '../../stitches.config.ts';
 
```

Comment:
Moving this file caused a build failure on all the files dependent on it, could we fix those imports in this PR?

![image](https://user-images.githubusercontent.com/7143571/148659745-131c7199-c9e8-4726-8cd6-d5b3e80c693f.png)